docs/source/_ext/compatibilitytable.py

```diff
@@ -0,0 +1,102 @@
+import os.path
+
+import yaml
+from docutils import nodes
+from docutils.parsers.rst import Directive
+
+
+def has_helm_chart(table: dict, component: str) -> bool:
```

Comment:
Do you mean in the YAML source or in the Python file?

I'm not too keen on that either way, because it's information duplication (it can be deduced from the data relatively easily, and I'd rather the computer do it than I)

Reply:
I see, in my point of view it was a way to ensure that the info is present in the data. But I don't have a strong opinion, I'll approve then ! thx for the pr this will be more than useful !!